runners/genlayer-py-std/src-emb/genlayer_embeddings/_nn/builder.py

```
@@ -37,24 +78,61 @@ def add_decl(self, expr) -> str:
-		# arr.setflags(write=False)
-		self._globals[name] = v
+		if isinstance(v, np.ndarray):
+			v = v.copy()
+
+			as_bytes = v.tobytes()
+			if len(as_bytes) > 128:
+				if self._compress:
+					import zlib
+
+					new_bytes = zlib.compress(as_bytes, level=9)
+					as_str = f'zlib.decompress(base64.b64decode({repr(base64.b64encode(new_bytes).decode("ascii"))}))'
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)